pyspecProcScripts/correlation_alignment.py

```diff
@@ -152,19 +152,19 @@ def correl_align(s_orig, align_phases=False,tol=1e-4,indirect_dim='indirect',
         s_orig.ft(direct)
         s_copy.ft(direct)
         if my_iter == 0:
-            logging.info(strm("holder"))
+            logging.debug(strm("holder"))
             if fl:
                 fl.image(s_copy.C.setaxis(indirect_dim,'#').set_units(indirect_dim,'scan #'),
                         ax=ax_list[3],human_units=False)
                 ax_list[3].set_title('after correlation\nbefore ph0 restore')
-        logging.info(strm('signal energy per transient (recalc to check that it stays the same):',(abs(s_copy**2).data.sum().item() / N)))
+        logging.debug(strm('signal energy per transient (recalc to check that it stays the same):',(abs(s_copy**2).data.sum().item() / N)))
 
         this_E = (abs(s_copy.C.sum(indirect_dim))**2).data.sum().item() / N**2
         energy_vals.append(this_E / sig_energy)
-        logging.info(strm('averaged signal energy (per transient):', this_E))
+        logging.debug(strm('averaged signal energy (per transient):', this_E))
         if last_E is not None:
             energy_diff = (this_E - last_E)/sig_energy
-            logging.info(strm(energy_diff))
+            logging.debug(strm(energy_diff))
```

Comment:
again, this actually came from building sphinx -- I see all this noise printed on the gallery page, and so I get rid of it.